src/content/docs/accounts/accounts-billing/account-structure/multi-tenancy/intro-to-multi-tenancy.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "Introduction to multi-tenancy" 
+metaDescription: "The multi-tenancy feature helps you manage multiple organizations and accounts"
+freshnessValidatedDate: 2024-09-05
+---
+
+If your company is composed of more than one business unit, you can use multi-tenancy in New Relic to model your company structure. This feature allows you to set up multiple tenants (organizations) related to a primary, managing organization. 
+
+This structure prevents the leakage of personally identifiable information (PII) and sets secure data boundaries between business units or end customers. You'll benefit from these boundaries whether you have a large enterprise company with multiple, independent business units or have a managed service provider (MSP) overseeing customer environments.
```

Comment:
```suggestion
This structure prevents the leakage of personally identifiable information (PII) and sets secure data boundaries between business units or end customers. You'll benefit from these boundaries whether you have a large enterprise company with multiple, independent business units, or a managed service provider (MSP) overseeing customer environments.
```

Reply:
@ally-sassman I'll go ahead and drop the "have" but I'll leave out the comma since it's just "this or that."